src/vs/editor/contrib/suggest/browser/suggestWidget.ts

```diff
@@ -30,7 +30,7 @@ import { attachListStyler } from 'vs/platform/theme/common/styler';
 import { isHighContrast } from 'vs/platform/theme/common/theme';
 import { IColorTheme, IThemeService } from 'vs/platform/theme/common/themeService';
 import { CompletionModel } from './completionModel';
-import { ResizableHTMLElement } from './resizable';
+import { ResizableHTMLElement } from '../../../../base/browser/ui/resizable/resizable';
```

Comment:
ts-refactor?

Reply:
Yep, I guess it keeps it relative if it already was